com.playonlinux.PlayOnLinux4.yaml

```diff
@@ -0,0 +1,147 @@
+id: com.playonlinux.PlayOnLinux4
```

Comment:
Any reason to include version number in the ID?

Reply:
So it don't get confused with [PlayOnLinux 5](https://github.com/PhoenicisOrg/phoenicis), which is already on Flathub. It sounds just like a newer Version, but it is a completely different Program that was written from Scratch in a different Programming Language. Save data and Backups are not compatible. The [name of the Repo](https://github.com/PlayOnLinux/POL-POM-4) is also `POL-POM-4`.

Reply:
That's fair, I forgot we already have POL5 on Flathub.